sample/src/main/java/vehicles/AbstractVehicle.java

```diff
@@ -0,0 +1,40 @@
+package vehicles;
+
+import people.Person;
+
+import java.time.LocalDate;
+
+/**
+ * An abstract class for implementing different kinds of {@link Vehicle}.
```

Comment:
If you feel strongly about this, please stick to your guns.  Discourse is one driving force behind high quality.  Could you implement this link conversion in a separate PR to keep this one focused?